src/test/java/org/apache/commons/collections4/comparators/ComparatorChainTest.java

```diff
@@ -67,11 +68,8 @@ public void testBadNoopComparatorChain() {
         final ComparatorChain<Integer> chain = new ComparatorChain<>();
         final Integer i1 = 4;
         final Integer i2 = 6;
-        try {
-            chain.compare(i1, i2);
-            fail("An exception should be thrown when a chain contains zero comparators.");
-        } catch (final UnsupportedOperationException e) {
-        }
+
+        assertThrows(UnsupportedOperationException.class, () -> chain.compare(i1, i2));
```

Comment:
I think the error message would be helpful in this case :point_up: maybe add that to the `assertThrows`

Reply:
ditto too